codalab/model/tables.py

```diff
@@ -78,7 +79,7 @@
   Column('uuid', String(63), nullable=False),
   Column('name', String(255), nullable=False),
   Column('owner_id', String(255), nullable=True),
-  Column('title', String(255), nullable=True),  # Short human-readable description of the worksheet
+  Column('title', Unicode(255), nullable=True),  # Short human-readable description of the worksheet
```

Comment:
`Text` is not unicode by default, `UnicodeText` is. Which is why I think there is some manual conversion of types. I would try to change more of the columns to `Unicode*` but I'm not prepared to test all those changes..